### PR TITLE
Inner transactions in operations index

### PR DESCRIPTION
**Pull request by miiu96**
- Added inner transactions in the 'operations' index
- The inner transactions from a relayed v3 transaction now are indexed as a separate object with a new type
- PR from `mx-chain-core-go`: https://github.com/multiversx/mx-chain-core-go/pull/330

### CODE REVIEW

File: process/elasticproc/elasticProcessor.go [hunk 1]
```diff
@@ -56,6 +56,7 @@ type ArgElasticProcessor struct {
 	DBClient           DatabaseClientHandler
 	LogsAndEventsProc  DBLogsAndEventsHandler
 	OperationsProc     OperationsHandler
+	InnerTxsHandler    InnerTxsHandler
```
Comment (sstanculeanu) [inline process/elasticproc/elasticProcessor.go:59]: missing nil check

Comment (miiu96) [reply process/elasticproc/elasticProcessor.go:59]: the check for nil is the `check.go` file and it's done on line 82.

**Review by sstanculeanu — DISMISSED** [review]

**Review by ssd04 — DISMISSED** [review]

**Review by sstanculeanu — APPROVED** [review]

**Review by ssd04 — APPROVED** [review]

**Review by sstanculeanu — APPROVED** [review]

**Review by ssd04 — APPROVED** [review]

File: integrationtests/testdata/relayedTxV3/relayed-v3-no-refund.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      "receiver": "erd1aduqqezzw0u3j7tywlq3mrl0yn4z6f6vytdju8gg0neq38fauyzsa5yy6r",
+      "sender": "erd10ksryjr065ad5475jcg82pnjfg9j9qtszjsrp24anl6ym7cmeddshwnru8",
+      "gasPrice": 1000000000,
+      "gasLimit": 60000000,
+      "data": "ZG9Tb21ldGhpbmc=",
+      "chainID": "",
+      "version": 0
+    }
+  ]
+}
```
Comment (sstanculeanu) [inline integrationtests/testdata/relayedTxV3/relayed-v3-no-refund.json:58]: missing new line

**Review by sstanculeanu — DISMISSED** [review]

File: integrationtests/testdata/relayedTxV3/relayed-v3-no-refund.json (hunk 2 above)
Comment (miiu96) [reply integrationtests/testdata/relayedTxV3/relayed-v3-no-refund.json]: added

**Review by sstanculeanu — DISMISSED** [review]

**Review by ssd04 — DISMISSED** [review]

**Review by sstanculeanu — APPROVED** [review]

**Pull request closed without merging**